runtime/runc_jailer.go

```diff
@@ -287,52 +326,6 @@ func (j *runcJailer) BuildJailedRootHandler(cfg *config.Config, machineConfig *f
 			}
 
 			j.logger.Info("Successfully ran jailer handler")
-			j.started = true
```

Comment:
Why do you want to move `j.started = true` from here to the handler above? I'd like to keep this line here, since we may reorder the handlers in future.

Reply:
`j.started = true` in the `BuildJailedRootHandler` will block `BuildBindMountFifoHandler`, since it is always the first handler. Once it set up `j.started=true` which means the runcJailer is started, no bind mount is allowed as [this line of code](https://github.com/firecracker-microvm/firecracker-containerd/blob/7e9026aae9099b8e8be7c0a1acd20ea04c5aab93/runtime/runc_jailer.go#L424) is shown. `BuildBindMountFifoHandler` is triggered after `firecracker.CreateLogFilesHandlerName`.

Reply:
Hmm, how does that work? bindMountFileToJail is modifying runc's configuration. It has to be executed before starting runc.

Reply:
If `j.started=true` is placed inside `BuildBindMountFifoHandler`, it will work. I thought the runc started after all the defaultHandlers. So I just moved the line inside `BuildBindMountFifoHandler`. Now I am not quite sure when the runc is started.